src/python/Dockerfile

```diff
@@ -0,0 +1,487 @@
+RUN source venv/bin/activate && \
+    cd grapl-tests-common && \
+    pip install .
+
+
+#
+# dgraph-ttl
+#
+
+# build
+FROM grapl-python-build AS dgraph-ttl-build
+
+COPY --chown=grapl python/grapl-dgraph-ttl dgraph-ttl
+COPY --chown=grapl --from=grapl-analyzerlib-build /home/grapl/venv venv
+
+RUN source venv/bin/activate && \
+    pip install -r dgraph-ttl/requirements.txt
+
+# zip
+FROM dgraph-ttl-build AS dgraph-ttl-zip
+
+RUN ORIG_DIR=$(pwd); \
+    cd ~/venv/lib/python3.7/site-packages && \
+    zip -q9r -dg "${ORIG_DIR}/lambda.zip" ./ && \
+    cd ~/dgraph-ttl && \
+    zip -g "${ORIG_DIR}/lambda.zip" app.py
+
+# deploy
+FROM grapl-python-deploy AS dgraph-ttl-deploy
+
+COPY --chown=grapl --from=dgraph-ttl-build /home/grapl/venv venv
+COPY --chown=grapl --from=dgraph-ttl-build /home/grapl/dgraph-ttl app
+
+CMD source venv/bin/activate && \
+    cd app && \
+    chalice local --no-autoreload --host=0.0.0.0 --port=8124
+
+# test
+FROM dgraph-ttl-build AS dgraph-ttl-test
+
+## Install test requirements
+COPY --chown=grapl --from=python-test-deps /home/grapl/python_test_deps python_test_deps
+RUN python_test_deps/install_requirements.sh
+
+
+#
+# model-plugin-deployer
+#
+
+# build
+FROM grapl-python-build AS model-plugin-deployer-build
+
+COPY --chown=grapl python/grapl-model-plugin-deployer model-plugin-deployer
+COPY --chown=grapl --from=grapl-analyzerlib-build /home/grapl/venv venv
+
+RUN source venv/bin/activate && \
+    cd model-plugin-deployer && \
+    pip install .
+
+# zip
+FROM model-plugin-deployer-build AS model-plugin-deployer-zip
+
+RUN ORIG_DIR=$(pwd); \
+    cd ~/venv/lib/python3.7/site-packages && \
+    zip -q9r -dg "${ORIG_DIR}/lambda.zip" ./ && \
+    cd ~/model-plugin-deployer/src && \
+    find . -type f -name "*.py" -exec zip -g "${ORIG_DIR}/lambda.zip" "{}" \;
+
+# deploy
+FROM grapl-python-deploy AS model-plugin-deployer-deploy
+
+COPY --chown=grapl --from=model-plugin-deployer-build /home/grapl/venv venv
+
+RUN source venv/bin/activate && \
+    chalice new-project app/
+
+COPY --chown=grapl --from=model-plugin-deployer-build /home/grapl/model-plugin-deployer/src/grapl_model_plugin_deployer.py app/app.py
+
+CMD source venv/bin/activate && \
+    cd app && \
+    chalice local --no-autoreload --host=0.0.0.0 --port=8123
+
+# test
+FROM model-plugin-deployer-build AS model-plugin-deployer-test
+
+## Install test requirements
+COPY --chown=grapl --from=python-test-deps /home/grapl/python_test_deps python_test_deps
+RUN python_test_deps/install_requirements.sh
+
+
+#
+# swarm-lifecycle-event-handler
+#
+
+# build
+FROM grapl-python-build AS swarm-lifecycle-event-handler-build
+
+COPY --chown=grapl python/swarm-lifecycle-event-handler swarm-lifecycle-event-handler
+
+RUN source venv/bin/activate && \
+    pip install -r swarm-lifecycle-event-handler/requirements.txt
+
+# zip
+FROM swarm-lifecycle-event-handler-build AS swarm-lifecycle-event-handler-zip
+
+RUN ORIG_DIR=$(pwd); \
```

Comment:
It'd be more clear to call this something like PACKAGE_NAME rather than ORIG_DIR; the latter primes you to think that we're capturing a directory to ultimately change back into once we're done doing whatever we need to do.

Reply:
Sure, yeah, makes sense. I'm going with 'LAMBDA_DIR_NAME' instead of PACKAGE_NAME.